### PR TITLE
better name lengths

### PR DESCRIPTION
**Pull request by calellowitz**
@millerdev cc: @sravfeyn

### CODE REVIEW

**Comment by calellowitz** [comment]
Not totally convinced this is the issue so holding off for now

**Comment by calellowitz** [comment]
problem resolved. this will fix it

File: corehq/warehouse/migrations/0029_name_length.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        ),
+        migrations.AlterField(
+            model_name='userdim',
+            name='last_name',
+            field=models.TextField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='userstagingtable',
+            name='first_name',
+            field=models.CharField(null=True),
```
Comment (emord) [inline corehq/warehouse/migrations/0029_name_length.py]: should probably recreate the migration so you don't miss these :)

**Review by emord — APPROVED** [review]
To confirm, these tables are empty right? Otherwise I think this will be a long migration that might hang

**Comment by calellowitz** [comment]
Confirmed. The initial load is what failed

**Review by millerdev — APPROVED** [review]

**Comment by calellowitz** [comment]
rolling in some other warehouse work i wanted to do that needs testing, so adding back the do not merge tag

**Comment by calellowitz** [comment]
good to go

File: corehq/warehouse/migrations/0030_one_synclog_per_user.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    dependencies = [
+        ('warehouse', '0029_name_length'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='synclogfact',
+            name='user_dim',
+            field=models.OneToOneField(on_delete=django.db.models.deletion.PROTECT, to='warehouse.UserDim'),
+        ),
```
Comment (emord) [inline corehq/warehouse/migrations/0030_one_synclog_per_user.py]: what does this sort of migration look like in SQL?

**Review by emord — APPROVED** [review]

**Pull request merged**